FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/logging-stack/logging_levels.h

```diff
@@ -0,0 +1,114 @@
+/*
+ * FreeRTOS Kernel V10.3.0
```

Comment:
Incorrect library name!

Reply:
Yea. Thanks

Reply:
All of our other files in the demo directory have this for the version. I think it may have been decided during our last release. If there aren't any plans to change it, then we may want to revert this file to how it was before